README.md

```diff
@@ -227,18 +343,121 @@ NxtPipeline::Pipeline.new do |p|
 end
 ```
 
-Note that the `after_execute` callback will not be called in case a step raises an error. 
+Note that the `after_execute` callback will not be called in case a step raises an error.
 See the previous section (_Error callbacks_) for how to define callbacks that run in case of errors.
 
-### Step resolvers
+### Constructor resolvers
```

Comment:
I'm ok keeping them here if you like them, but generally I'm still questioning whether having constructor (step) resolvers is worth it. When we had them in one of the services I found them confusing and not worth the overhead.

Reply:
I still think it makes a lot of sense :-D